src/allmydata/client.py

```diff
@@ -601,9 +598,9 @@ def init_magic_folder(self):
                     client=self,
                     upload_dircap=mf_config["upload_dircap"].encode('ascii'),
                     collective_dircap=mf_config["collective_dircap"].encode('ascii'),
-                    local_path_u=abspath_expanduser_unicode(local_dir_config, base=self.basedir),
+                    local_path_u=self.config.get_config_path(local_dir_config),
```

Comment:
And another

Reply:
hmm, maybe it's safer/easier to just make `get_config_path` re-expand every time